addon/components/image-layer.js

```diff
@@ -1,19 +1,43 @@
+import Ember from 'ember';
 import BaseLayer from 'ember-leaflet/components/base-layer';
 
+const {
+  deprecate,
+  isPresent
+} = Ember;
+
 export default BaseLayer.extend({
 
   leafletRequiredOptions: [
-    'imageUrl', 'bounds'
+    'url', 'bounds'
   ],
 
   leafletOptions: [
     'attribution', 'opacity'
   ],
 
   leafletProperties: [
-    'url', 'opacity'
+    'url', 'opacity', 'bounds'
   ],
 
+  init() {
+    const imageUrl = this.get('imageUrl');
```

Comment:
Some more info on why this usage of let/const: https://madhatted.com/2016/1/25/let-it-be